exporter/awsemfexporter/emf_exporter.go

```diff
@@ -45,17 +43,15 @@ const (
 )
 
 type emfExporter struct {
-	// Each (log group, log stream) keeps a separate pusher because of each (log group, log stream) requires separate stream token.
-	groupStreamToPusherMap map[string]map[string]cwlogs.Pusher
-	svcStructuredLog       *cwlogs.Client
-	config                 config.Exporter
-	logger                 *zap.Logger
+	svcStructuredLog *cwlogs.Client
+	config           config.Exporter
+	logger           *zap.Logger
 
 	metricTranslator metricTranslator
 
-	pusherMapLock sync.Mutex
-	retryCnt      int
-	collectorID   string
+	retryCnt    int
+	collectorID string
```

Comment:
What does the collector identifier to here?

Reply:
it is used for the log stream name variable. it is a possible value